app/routes/v2/clusters.js

```diff
@@ -215,7 +217,24 @@ const updateClusterResources = async (req, res, next) => {
             Stats.updateOne({ org_id: req.org._id }, { $inc: { deploymentCount: 1 } }, { upsert: true });
           }
 
-          await Resources.updateOne(key, changes, options);
+          const result = await Resources.updateOne(key, changes, options);
+          // publish notification to graphql
+          if (process.env.AUTH_MODEL && result) {
```

Comment:
See comment above about not using `AUTH_MODEL` for checking if graphql is enabled.

Reply:
Add ENABLE_GRAPHQL environment variable , default is false. see [here](https://github.com/razee-io/Razeedash-api/blob/graphql/app/index.js#L87) and [here](https://github.com/razee-io/Razeedash-api/blob/graphql/app/routes/v2/clusters.js#L243)